README.md

```diff
@@ -1 +1,139 @@
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <h3><b>Stock-Metrics-WebApp</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Stock-Metrics-WebApp <a name="Stock-Metrics-WebApp"></a>
+
+Companies is a React and Redux SPA application that displays interesting information about various companies in the world. It fetchs all the information that it displays using API request to a server.
+
+## 🛠 Built With <a name="built-with"></a>
+
+- React
+- Redux
+- Git, Github
+- VSCode
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Git,
+> Github,
+> React
+> Redux
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.w3schools.com/react/">React</a></li>
+
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- Basic set up
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+Clone this repository to your desired folder:
+
+```
+git clone https://github.com/Ashnaali3255/Stock-Metrics-WebApp.git
+```
+
+### Prerequisites
+
+- NodeJS
+- React & Redux
+- Browser
+
+In order to run this project you need:
+
+### Setup
+
+Clone this repository to your dist folder:
+
+git clone: https://github.com/Ashnaali3255/Stock-Metrics-WebApp.git
+
+### Install
+
+npm install
+
+### Usage
+
+Use this project in the browsers of desktops and mobile devices
+
+### Run tests
+
+npm run test
+
+### Deployment
+
+You can deploy this project on GitHub Pages
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Ashna Ali**
+
+- GitHub: [@githubhandle](https://github.com/Ashnaali3255)
+- Twitter: [@twitterhandle](https://twitter.com/Ashna_Ali1)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/ashna-ali-342151255/)
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+Add more dynamic features of sending data to user's email
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, please let me know and we can improve it further. You are welcome to support this project by giving suggestions for improvement.
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse.
+
```

Comment:
- [ ] As required for the project, kindly ensure that The `Acknowledgements` section in your README you give appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).